services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/client/StatsS3ReadingConfig.java

```diff
@@ -0,0 +1,40 @@
+package app.coronawarn.server.services.distribution.objectstore.client;
+
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig.StatisticsConfig;
+import java.net.URI;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.retry.annotation.EnableRetry;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+
+@Configuration
+@EnableRetry
+public class StatsS3ReadingConfig {
+
+  @Bean(name = "stats-s3")
+  public ObjectStoreClient createObjectStoreClient(DistributionServiceConfig distributionServiceConfig) {
+    return createClient(distributionServiceConfig.getStatistics());
+  }
+
+  private ObjectStoreClient createClient(StatisticsConfig s3stats) {
+    AwsCredentialsProvider credentialsProvider = StaticCredentialsProvider.create(
+        AwsBasicCredentials.create(s3stats.getAccessKey(), s3stats.getSecretKey()));
+    String endpoint = removeTrailingSlash(s3stats.getEndpoint());
+
+    return new S3ClientWrapper(S3Client.builder()
+        .region(Region.of("region-1"))
```

Comment:
Should this be a constant. I see in other configs we have: 
private static final Region DEFAULT_REGION = Region.EU_CENTRAL_1;

Reply:
Agreed. We can create a new constant for it. It's not `Region.EU_CENTRAL_1` though. It's really a region string of `region-1`...